docs/release-notes/release-notes-0.18.0.md

```diff
@@ -0,0 +1,53 @@
+# Release Notes
+
+* [Bug Fixes](#bugs)
+* [New Features](#new)
+  * [Functional Enhancements](#newfeature)
+  * [RPC Additions](#newrpc)
+  * [lncli Additions](#newlncli)
+* [Improvements](#update)
+  * [Functional Updates](#updatedfeature)
+  * [Updated RPCs](#updatedrpc)
+  * [Updated lncli](#updatedlncli)
+  * [Breaking Changes](#breakingchange)
+  * [Performance Improvements](#performance)
+* [Technical/Architectural updates](#tech)
+  * [BOLT Spec Updates](#boltspec)
+  * [Testing](#testing)
+  * [Database](#db)
+  * [Code Health](#codehealth)
+  * [Tooling and Documentation](#toolinganddocs)
+
+## <a name="bugs"></a>Bug Fixes
```

Comment:
Just FYI: These `<a name=...>` tags aren't strictly needed. By default the Markdown renderer will create those from the titles automatically, if you just take everything lowercase and replace spaces with dashes. So this would automatically be linkable as `[example](#bug-fixes)`.

Reply:
updated per your suggestion